robottelo/helpers.py

```diff
@@ -36,6 +36,29 @@ def get_server_software():
     return 'downstream' if return_code == 0 else 'upstream'
 
 
+def get_server_version():
+    """Read Satellite version.
+
+    Inspect server
+    /usr/share/foreman/lib/satellite/version.rb in order
+    to get the installed Satellite version.
+
+
+    :return: Either a string containing the Satellite
+        version or ``None`` if the version.rb file is not
+        present.
```

Comment:
nitpick: this can fit in two lines:
```py
        :return: Either a string containing the Satellite version or
            ``None`` if the version.rb file is not present.
```